src/performance/catch/CatchScore.cpp

```diff
@@ -69,7 +69,7 @@ CatchScore::CatchScore(
 			_value *= 1.01f + 0.04f * (11.0f - std::min(11.0f, approachRate)); // 5% at AR 10, 1% at AR 11
 
 		if (approachRate <= 9.0f)
-			_value *= 1.0f + 0.04f * (8.0f - approachRate); // Additional 4% for each AR below 9
+			_value *= 1.0f + 0.04f * (approachRate - 8.0f); // Additional 4% for each AR below 9
```

Comment:
This is the correct one, my mistake.
```suggestion
		if (approachRate < 9.0f)
			_value *= 1.0f + 0.04f * (9.0f - approachRate); // Additional 4% for each AR below 9
```